src/Swarrot/Processor/Ack/AckProcessor.php

```diff
@@ -87,12 +88,10 @@ private function handleException($exception, Message $message, array $options)
         $this->messageProvider->nack($message, $requeue);
 
         $this->logger->error(
-            sprintf(
-                '[Ack] An exception occurred. Message #%d has been %s.',
-                $message->getId(),
-                $requeue ? 'requeued' : 'nack\'ed'
-            ),
+            '[Ack] An exception occurred.',
```

Comment:
Maybe we should keep the part about `Message has been requeued|nack'ed.`. There will only be 2 alternatives in the grouping.

Reply:
Yeah maybe :thinking:

Reply:
I will create a separate PR for both issues

Reply:
See #192